wrapanapi/systems/virtualcenter.py

```diff
@@ -228,6 +258,17 @@ def _pick_datastore(self, allowed_datastores):
             raise Exception("No possible datastores!")
         return possible_datastores[0]
 
+    def _pick_datastore_cluster(self):
+        """Pick a datastore cluster based on free space."""
+        possible_datastores = [dsc for dsc in self.system.get_obj_list(vim.StoragePod)
+                               if dsc.overallStatus != "red"]
+        possible_datastores.sort(
+            key=lambda dsc: float(dsc.summary.freeSpace) / float(dsc.summary.capacity),
+            reverse=True)
+        if not possible_datastores:
+            raise Exception("No possible datastore clusters!")
```

Comment:
Optional: maybe include some information as to why the clusters are not available for use? e.g. say their overallStatus is `red`.

Reply:
:+1: